### DISCUSSION TITLE
Support a light theme for the header on github.com

### DISCUSSION THREAD

**Discussion by hwittenborn** [community/community#4406] [Other Feature Feedback, Questions, & Ideas]
Pretty much everything on the GitHub website supports either a light and dark mode, except for the header:

---
![image](https://user-images.githubusercontent.com/74838472/123859273-128b5480-d8ea-11eb-87c0-6a8043558287.png)

---

It would be nice if I could set this to a light theme in addition to everything else being on a light theme.

**Comment by gaknoll** [comment]
Thanks for the feedback @hwittenborn! I'll share this with the team working on our themes and navigation experiences.
